conda_lock/conda_lock.py

```diff
@@ -385,9 +385,12 @@ def make_lock_files(
             )
         platforms_to_lock = sorted(set(platforms_to_lock))
 
-        if platforms_to_lock:
+        if not platforms_to_lock:
+            new_lock_content = original_lock_content
```

Comment:
Since previously we were mutating `lock_content` in this conditional block, the "else" condition was lack of mutation. I think making that case explicit and first made this renaming change the most readable.